tests/Rector/Contrib/Sylius/Review/ReplaceCreateMethodWithoutReviewerRectorTest.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Rector\Contrib\Sylius\Review;
+
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+/**
+ * @covers \Rector\Rector\Contrib\Sylius\Review\ReplaceCreateMethodWithoutReviewerRector
```

Comment:
Do we need this annotation?

Reply:
Is there a better way to get to the class from test on 1 click?